src/screens/MemberDetail/MemberDetail.tsx

```
@@ -57,207 +48,138 @@ const MemberDetail: React.FC<MemberDetailProps> = ({ id }): JSX.Element => {
   const isMounted = useRef(true);
   const { getItem, setItem } = useLocalStorage();
   const [show, setShow] = useState(false);
-  const currentUrl = location.state?.id || getItem('id') || id;
-
-  const { orgId } = useParams();
-  const navigate = useNavigate();
-
-  const [unassignUserTagModalIsOpen, setUnassignUserTagModalIsOpen] =
-    useState(false);
+  const currentId = location.state?.id || getItem('id') || id;
+  const [selectedAvatar, setSelectedAvatar] = useState<File | null>(null);
```

ESLint

[error] 'setSelectedAvatar' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)